package.json

```diff
@@ -4,8 +4,13 @@
   "description": "Some sample web app.",
   "main": "index.html",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "live-server"
+    "build-task:scss-compile": "node-sass-chokidar --source-map true _scss/style.scss -o _temp",
+    "build-task:autoprefixer": "postcss _temp/style.css --use autoprefixer -d _css",
+    "sass:build": "npm-run-all -p build-task:*",
+    "sass:watch": "chokidar '_scss/**/*.scss' -c 'npm run sass:build'",
```

Comment:
`sass:watch` Uses the `chokidar` file watcher package to detect changes that happen to any .scss file within the _scss directory, and upon changes, runs `npm run sass:build`.